src/components/suggest/suggest.test.tsx

```diff
@@ -138,6 +138,25 @@ describe('EuiSuggest', () => {
       });
     });
 
+    describe('onItemClick', () => {
+      it('passes an onChange callback to the underlying EuiSelectable, which will fire on list item clicks and enter keypresses', () => {
+        const onItemClick = jest.fn();
+        const component = shallow(
+          <EuiSuggest
+            {...requiredProps}
+            suggestions={sampleItems}
+            onItemClick={onItemClick}
+          />
+        );
+
+        const options: any = component.find('EuiSelectable').prop('options');
```

Comment:
Enzyme allows `find`ing by a component variable itself instead of the name, and automatically discovers the shape of that component's props:

If `EuiSelectable` is imported in this file, this line can use the variable and drop the `any`

```suggestion
        const options = component.find(EuiSelectable).prop('options');
```

Reply:
Super cool!!! I haven't used that with Typescript previously, TIL ✨ https://github.com/elastic/eui/pull/5693/commits/c665b3aa094f2ca771c9fb8af1c71ecf782a44b8